docs/antrea-network-policy.md

```diff
@@ -10,6 +10,9 @@
   - [kubectl commands for Tier](#kubectl-commands-for-tier)
 - [Antrea ClusterNetworkPolicy](#antrea-clusternetworkpolicy)
   - [The Antrea ClusterNetworkPolicy resource](#the-antrea-clusternetworkpolicy-resource)
+    - [ACNP with stand alone selectors](#acnp-with-stand-alone-selectors)
+    - [ACNP with ClusterGroup reference](#acnp-with-clustergroup-reference)
+    - [ACNP with Namespaces](#acnp-with-namespaces)
```

Comment:
Yes that's what I had in mind, although maybe we could name that one after the use case "ACNP for default Namespace isolation". What do you think?

Reply:
done